packages/peregrine/lib/talons/CheckoutPage/useCheckoutPage.js

```diff
@@ -56,31 +52,33 @@ export const useCheckoutPage = props => {
 
     const [
         getCheckoutDetails,
-        { data: checkoutData, loading: checkoutLoading }
-    ] = useLazyQuery(getCheckoutDetailsQuery, {
-        // TODO: Purposely overfetch and hit the network until all components
-        // are correctly updating the cache. Will be fixed by PWA-321.
-        fetchPolicy: 'cache-and-network'
-    });
+        {
+            data: checkoutData,
+            called: checkoutCalled,
+            client,
+            loading: checkoutLoading
+        }
+    ] = useLazyQuery(getCheckoutDetailsQuery);
 
-    const [getCheckoutStep, { data: stepData, client }] = useLazyQuery(
-        getCheckoutStepQuery
```

Comment:
I replaced this query with a client side resolver for the `checkoutStep` property of `Cart` types.